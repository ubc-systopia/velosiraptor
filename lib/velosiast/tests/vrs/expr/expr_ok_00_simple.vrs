1 + 1;
2 * 2;
1 + 2 * 3;
1 * 2 + 3;
1 == 1;
a == 1;
1 == a;
a == 1;
// a == a;
(a > b) == true;