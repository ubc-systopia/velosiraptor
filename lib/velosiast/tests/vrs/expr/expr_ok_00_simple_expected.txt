0x2

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

0x4

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

0x7

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

0x5

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

true

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

a == 0x1

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

a == 0x1

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

a == 0x1

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

(b < a) == true

@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@

